handlers/handlers.go

```diff
@@ -40,68 +42,98 @@ func setStatusCode(req *http.Request, w http.ResponseWriter, err error) {
 	w.WriteHeader(status)
 }
 
-func forwardFlorenceTokenIfRequired(req *http.Request) *http.Request {
-	if len(req.Header.Get(common.FlorenceHeaderKey)) > 0 {
-		ctx := common.SetFlorenceIdentity(req.Context(), req.Header.Get(common.FlorenceHeaderKey))
-		return req.WithContext(ctx)
-	}
-	return req
+func setStatusCode404(req *http.Request, w http.ResponseWriter, err error) {
+	status := http.StatusNotFound
+	log.ErrorCtx(req.Context(), err, log.Data{"setting-response-status": status})
+	w.WriteHeader(status)
 }
 
-//GeographyRender ...
-func GeographyRender(rend RenderClient) http.HandlerFunc {
+//GeographyHomepageRender ...
+func GeographyHomepageRender(rend RenderClient) http.HandlerFunc {
 
 	return func(w http.ResponseWriter, req *http.Request) {
 		var page geographyHomepage.Page
 
-		resp, err := http.Get(`https://api.dev.cmd.onsdigital.co.uk/v1/code-lists` + localAuthority)
+		homePageLink := url + `/code-lists` + localAuthority
```

Comment:
Sorry nit-picky comment: local auth is already defined as a constant - you could also define `/code-lists` as a constant and then: `fmt.SprintF("%s%s%s", url, codelistConst, localAuthority)`